ipycytoscape/cytoscape.py

```diff
@@ -109,7 +109,7 @@ class Graph(Widget):
 
     nodes = MutableList(Instance(Node)).tag(sync=True, **widget_serialization)
     edges = MutableList(Instance(Edge)).tag(sync=True, **widget_serialization)
-
+    directed = Bool().tag(sync=True, **widget_serialization)
```

Comment:
adding a new line will keep the spacing consistent
```suggestion
    directed = Bool().tag(sync=True, **widget_serialization)

```

I'm not sure that this needs to sync to frontend, there doesn't seem to be anything there that will use this information. Maybe this could be a normal python boolean and be set to false by default in `init`.

Reply:
From what I understand about the architecture of ipycytoscape, I do need to propagate this to the frontend, as the Widget doesn't have access to the graph object itself.

Reply:
Ah I was slightly confused sorry. There are three levels of objects at play here:
1. Graph
2. CytoscapeWidget
3. The javascript side

I was referring to not needing to sync to the javascript side of things (2 -> 3). You are correct that the CytoscapeWidget does need to know about it this variable somehow (1->3).

Reply:
Yep, adding a new line would be nice :)
Also, you don't need the `**widget_serialization` here, just the `.tag(sync=True)`.